### DISCUSSION TITLE
关于提交验收问题

### DISCUSSION THREAD

**Discussion by bsnmldb** [liusy58/NJU_AdvancedProgramming_SP22#207] [Q&A]
提交后的程序如何重新打开呢
我将程序整体(包括代码 资源文件 pro)打包再打开打包的文件时(通过点击pro文件),项目无法编译
报错为`:-1: error: [Makefile:226: Makefile] Error 2`,是我再打开的方式有问题还是什么问题,如果出现这种问题,我是否只能在验收时下载文件,重新新建项目,复制文件,一个一个添加资源文件,再重新编译?
环境 Desktop Qt 6.2.4 MinGW 64-bit

**Comment by liusy58** [comment]
可以
